pages/partials/head.ejs

```diff
@@ -3,6 +3,9 @@
 <meta name="viewport" content="width=device-width, initial-scale=1">
 <title>PrairieLearn</title>
 <link href="/node_modules/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet" />
+<% if(locals.darkmode == 1){%>
+<link href="/node_modules/bootstrap/dist/css/bootstrap1.min.css" rel="stylesheet" />
```

Comment:
The `node_modules` directory is automatically created by `npm ci`, so we can't put things in there by hand. Extra themes should be installed by npm (preferably) or be put in `/public/stylesheets` (if necessary).